src/items/item.cpp

```diff
@@ -1502,6 +1521,24 @@ std::string Item::parseImbuementDescription(const Item* item)
 	return s.str();
 }
 
+std::string Item::parseClassificationDescription(const Item* item) {
+	std::ostringstream string;
+	if (item && item->getClassification() >= 1) {
```

Comment:
`if (item && item->getClassification() >= 0) {`

To show any item without tier classification.

Reply:
> `if (item && item->getClassification() >= 0) {`
> 
> To show any item without tier classification.

I believe that only item rated 1 or more should be shown.